style.css

```diff
@@ -504,6 +504,15 @@ textarea:valid {
   text-align: center;
 }
 
+.error-message {
+  color: #fff;
+  display: none;
+}
+
+.error-message.active {
```

Comment:
- [ ] Good job implementing the validations 👍 . Currently, your error messages are not properly aligned close to the submit button on the desktop view as specified in the requirement. Please fix it.

![image](https://user-images.githubusercontent.com/63473584/180342280-209c10cb-cdb8-422d-ad53-20e596cbe2cc.png)